examples/cell_tracking.py

```diff
@@ -0,0 +1,175 @@
+"""
+Updated Cell tracking example
+==============================
```

Comment:
```suggestion
Cell tracking
=============
```
"Updated" doesn't mean much to anyone else and this is our examples gallery, so I think example is redundant.

BTW It's critical that the number of = match the number of chars in the title. Right now docs build is failing because of the extra =

Reply:
Got it, edited "=" to match chars.